dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DvObjectServiceBean.java

```diff
@@ -260,7 +260,7 @@ private String getDvObjectIdListClause(List<Long> dvObjectIdList) {
                 " SELECT id, owner_id FROM dvobject WHERE id in (" + datasetIdStr + "))" +
                 " UNION\n" +
                 " SELECT o.id, o.owner_id FROM path_elements p, dvobject o WHERE o.id = p.owner_id) " +
-                "SELECT id, owner_id FROM path_elements WHERE owner_id IS NOT NULL;"; // ORDER by id ASC;";
+                "SELECT id, owner_id FROM path_elements";
```

Comment:
It would be nice to have some integration test for this method to check if it is working as we expect